src/operator/quantization/mkldnn/mkldnn_requantize-inl.h

```diff
@@ -99,55 +108,47 @@ static void MKLDNNRequantizeForward(const nnvm::NodeAttrs& attrs,
                                     const std::vector<NDArray>& outputs) {
   using namespace mshadow;
   using namespace mxnet_op;
+  using red::limits::MaxValue;
+  using red::limits::MinValue;
   typedef int32_t SrcDType;
   typedef int8_t  DstDType;
```

Comment:
This can be removed, right?

Reply:
It could. But it's not introduced by this PR, so it doesn't necessary to remove it in this PR.